dashboard/lib/dynamic_config/adapters/dynamodb_adapter.rb

```diff
@@ -0,0 +1,64 @@
+# A DynamoDB adapter class that allows us to use
+# dynamodb as a persistent store with the datastore_cache
+require 'aws-sdk'
+require 'oj'
+
+class DynamoDBAdapter
+  # @param table_name [String] the name of the dynamodb table to use
+  def initialize(table_name)
+    @table_name = table_name
+    @client = Aws::DynamoDB::Client.new
+  end
+
+  # @param key [String]
+  # @returns [String]
+  def get(key)
+    resp = @client.get_item({
+      table_name: @table_name,
+      key: {
+        'data-key' => key
+      }
+    })
+    return nil if resp.item.nil?
+    Oj.load(resp.item['data-value'])
+  end
+
+  # @param key [String]
+  # @param value [String]
```

Comment:
Yea, I was experimenting with moving the json serialization to a different layer.  I'll update the documentation